www/views/page/content/schrijfregels.php

```diff
@@ -1,21 +1,9 @@
+<?php
+/**
+ * @var string $schrijfregels
+ * */
+?>
 <h2>Schrijfregels</h2>
-<ul>
-	<li>Niet in de "ik" vorm schrijven, of jezelf anders noemen zoals de (razende) reporters of bij je naam.</li>
-	<li>Schrijf zo veel mogelijk in de tegenwoordige tijd.</li>
-	<li>Vermijdt woorden als "kunnen", "worden" en "zullen" die de zinnen onnodig langer maken.</li>
-	<li>Wist je datjes:<ul>
-		<li>Zet geen streepje of een ander teken voor de wistjedatjes en een vraagteken op het eind.</li>
-		<li>Beperk opeenvolgende wistjedatjes.</li>
-		<li>Minimaal ongeveer 5 per pagina.</li></ul>
-	<li>Schrijf begrijpbaar en lijk voor de desbetreffende speltak.</li>
-	<li>Hou er rekening mee dat alle speltakken alles kunnen lezen. Wil je foute teksten plaatsen bij de pivo's, doe dat dan zodanig dat de welpen die het lezen niet begrijpen.</li>
-	<li>Bedenk wat je zelf leuk zou vinden om te lezen of zien.</li>
-	<li>Schrijf geen komma, schrijf een punt. Vraag jezelf af of een komma echt nodig is, of of je er beter 2 losse zinnen van kan maken.</li>
-	<li>Zorg er voor dat stukjes kort zijn met een pakkende titel. Dat maakt het aantrekkelijker om meer te lezen dan alleen de wistjedatjes.</li>
-	<li>Zorg er voor dat in elk stukje een paar namen en hun scouting groep voorkomen. Ook bij thema stukjes (vraag bijvoorbeeld naar wat de mensen die er bij staan er van vinden).</li>
-	<li>Schijf leuke stukjes. Stukjes in de richting "post 1 was ...,  post 2 was  ... , etc." zijn niet zo leuk om te lezen.</li>
-	<li>Richtlijn is 500 tekens per stukje.</li>
-	<li>Getallen tot en met twintig schrijven we uit als woord (ipv 20).</li>
-	<li>Scoutinggroepen met een hoofdletter, bijvoorbeeld Scouting Veendam. Het woord "scouting" los is zonder hoofdletter.</li>
-	<li>Namen van hikes zijn met een hoofdlettter (Welpenhike, Scoutshike+, etc.).</li>
-</ul>
+<div class='schrijfregels'>
+<?php echo  $schrijfregels; ?>
```

Comment:
Dit is een [XSS risico](https://nl.wikipedia.org/wiki/Cross-site_scripting). Aangezien je wat ook maar kan invullen als schrijfregels kan iemand op deze manier alles injecteren op de schrijfregels pagina, inclusief vervelende `<script>`s en dergelijke.